user-guide/Basic_Functionality/Trending/Advanced_analytics_trending.md

```diff
@@ -4,9 +4,7 @@ uid: Advanced_analytics_trending
 
 # Advanced analytics in trend graphs
 
-## Local Advanced Analytics
```

Comment:
This has been removed again.
Does it make sense to have the 'Adding related parameters to a trend graph' entry in here that does not adhere to the explanation right before it?
- RL does not use SLAnalytics directly. Instead it uses the ModelHost DxM.
- It cannot be (de)activated via '*System Center* > *System settings* > *analytics config*'

Reply:
Adjusted - could you take another look?